.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules/
 package-lock.json
 yarn.lock
```

Comment:
Once again, with modern package managers, lockfiles should be committed to source control. Here are some resources elaborating on this topic:
 - https://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for
 - https://classic.yarnpkg.com/blog/2016/11/24/lockfiles-for-all/